docs/tutorial.rst

```diff
@@ -206,7 +206,9 @@ object::
         ross.last_name = 'Lawley'
         ross.save()
 
-Now that we've got our user in the database, let's add a couple of posts::
+It's your turn. Assign another user to the variable-name ``john``, just like we did above with ``ross``.
```

Comment:
I think this would sound more natural:
```
Assign another user to a variable called ``john``, just like we did above with ``ross``.
```

Also, please make sure to break the line so that each of the broken parts is shorter than 80 characters.